lib/omnibus/licensing.rb

```diff
@@ -25,6 +27,7 @@ class Licensing
     include Sugarable
 
     OUTPUT_DIRECTORY = "LICENSES".freeze
+    CACHE_DIRECTORY = "license-cache".freeze
```

Comment:
On second thought, never mind, it should be clear from the context. I just thought I was in a different file.